### PR TITLE
[PS 1081] Split Out API Methods into Cipher Admin API

### PR DESCRIPTION
**Pull request by cyprain-okeke**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [X] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective
Added abstractions for CipherAdminServiceAbstraction

Added implementations for CipherAdminServiceAbstraction in CipherAdminService

Updated all references to new CipherAdminServiceAbstraction

Deleted methods in CipherService abstraction and apiService that relates to Cipher Admin.

Fixed the import path for CipherAdminServiceAbstraction

## Code changes

- **libs/common/src/abstractions/cipher/cipher-admin.service.abstraction.ts:** added the file and moved admin cipher services from abstractions/api.service.ts to it.
- **libs/common/src/services/cipher/cipher-admin.service.ts:** added the file implementation for the abstractions/cipher-admin.service.abstraction.ts.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: apps/browser/src/background/main.background.ts [hunk 1]
```diff
@@ -271,16 +276,25 @@ export default class MainBackground {
     );
     this.settingsService = new SettingsService(this.stateService);
     this.fileUploadService = new FileUploadService(this.logService, this.apiService);
+
     this.cipherService = new CipherService(
       this.cryptoService,
       this.settingsService,
       this.apiService,
-      this.fileUploadService,
```
Comment (MGibson1) [inline apps/browser/src/background/main.background.ts:278]: Why is this included? if this work builds off of #3554, can you target that to try and separate review work?

If it doesn't depend on that PR, please make sure you're rebased to `master`

**Review by MGibson1 — COMMENTED** [review]
Please let me know when this is rebased or re-targeted.

File: apps/browser/src/background/main.background.ts (hunk 1 above)
Comment (cyprain-okeke) [reply apps/browser/src/background/main.background.ts]: I have re-targeted it
